From: Sven Eckelmann <sven@narfation.org>
Date: Mon, 23 Jan 2017 18:25:44 +0100
Subject: [PATCH] batman-adv: Increase aggregation time

Signed-off-by: Sven Eckelmann <sven@narfation.org>
---
 net/batman-adv/main.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/batman-adv/main.h b/net/batman-adv/main.h
index 5b78555..b839083 100644
--- a/net/batman-adv/main.h
+++ b/net/batman-adv/main.h
@@ -118,7 +118,7 @@
  * forw_packet->direct_link_flags
  */
 #define BATADV_MAX_AGGREGATION_BYTES 512
-#define BATADV_MAX_AGGREGATION_MS 100
+#define BATADV_MAX_AGGREGATION_MS 500
 
 #define BATADV_BLA_PERIOD_LENGTH	10000	/* 10 seconds */
 #define BATADV_BLA_BACKBONE_TIMEOUT	(BATADV_BLA_PERIOD_LENGTH * 6)
