From: root <root@fermi.chemnitz.freifunk.net>
Date: Mon, 26 Sep 2016 14:01:13 +0200
Subject: [PATCH] Make saving of nodes data save again
---
 backend.py | 18 ++++++++++++++----
 1 file changed, 14 insertions(+), 4 deletions(-)

diff --git a/backend.py b/backend.py
index 6800997..27335d0 100755
--- a/backend.py
+++ b/backend.py
@@ -142,18 +142,28 @@ def main(params):
     batadv_graph = graph.merge_nodes(batadv_graph)
     batadv_graph = graph.to_undirected(batadv_graph)
 
-    # write processed data to dest dir
-    with open(nodes_fn, 'w') as f:
+    # write processed data to temporary files in dest dir
+    # and rename them atomically to prevent inconsistent data
+    with open(nodes_fn + '.tmp', 'w') as f:
         json.dump(nodedb, f, sort_keys=True)
+        f.flush()
+        os.fsync(f.fileno())
+    os.rename(nodes_fn + '.tmp', nodes_fn)
 
     graph_out = {'batadv': json_graph.node_link_data(batadv_graph),
                  'version': GRAPH_VERSION}
 
-    with open(graph_fn, 'w') as f:
+    with open(graph_fn + '.tmp', 'w') as f:
         json.dump(graph_out, f, sort_keys=True)
+        f.flush()
+        os.fsync(f.fileno())
+    os.rename(graph_fn + '.tmp', graph_fn)
 
-    with open(nodelist_fn, 'w') as f:
+    with open(nodelist_fn + '.tmp', 'w') as f:
         json.dump(export_nodelist(now, nodedb), f, sort_keys=True)
+        f.flush()
+        os.fsync(f.fileno())
+    os.rename(nodelist_fn + '.tmp', nodelist_fn)
 
     # optional Graphite integration
     if params['graphite']:
