From: Sven Eckelmann <sven@narfation.org>
Date: Sat, 18 Mar 2017 20:33:51 +0100
Subject: [PATCH] Add raw wireless information from respondd
---
 lib/nodes.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/nodes.py b/lib/nodes.py
index c937ccc72308b63835ee8428b2120cd1db265dd5..b1980f5bbfad87acfec2edac2685e21141c91ba6 100644
--- a/lib/nodes.py
+++ b/lib/nodes.py
@@ -121,6 +121,7 @@ def import_statistics(nodes, stats):
         add(node, stats, 'traffic', ['traffic'])
         add(node, stats, 'gateway', ['gateway'])
         add(node, stats, 'gateway_nexthop', ['gateway_nexthop'])
+        add(node, stats, 'wireless_raw', ['wireless'])
         mark_uplink(node, stats)
 
 
