From: Sven Eckelmann <sven@narfation.org>
Date: Sun, 19 Mar 2017 10:03:57 +0100
Subject: [PATCH] Add txpower information from respondd
---
 lib/nodes.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/nodes.py b/lib/nodes.py
index ec5c45fc55cdb0a245a44dde6a9587d838ae6619..7a177c974b17e41724ed13ab8369e7324c40f3e9 100644
--- a/lib/nodes.py
+++ b/lib/nodes.py
@@ -127,6 +127,7 @@ def import_statistics(nodes, stats):
         add(node, stats, 'gateway', ['gateway'])
         add(node, stats, 'gateway_nexthop', ['gateway_nexthop'])
         add(node, stats, 'wireless_raw', ['wireless'])
+        add(node, stats, 'txpower', ['txpower'])
         mark_uplink(node, stats)
 
 
